tests/CMakeLists.txt

```diff
@@ -21,5 +21,9 @@ else()
   set(EXCLUDE_FROM_TESTS ${TESTDATA_TESTS})
 endif()
 
+if(NOT EXISTS ${FIVOXLFP_DIR}/lfpFunctor.h)
+  list(APPEND EXCLUDE_FROM_TESTS lfpValidation.cpp)
```

Comment:
It needs to combined also with the availability of TestData. That's why the Ubuntu build fails.